sentry/src/transports/ureq.rs

```diff
@@ -1,8 +1,7 @@
 use std::time::Duration;
 
-#[cfg(fdoc)]
```

Comment:
Blame says that I misspelled this :grimacing:

Reply:
In turn `` [`ureq`]: https://crates.io/crates/ureq `` below wouldn't have been needed...

Reply:
Looks like I fixed this in #455 already.